ocfweb/docs/docs/staff/tips/twitch.md

```diff
@@ -0,0 +1,35 @@
+[[!meta title="Using Twitch and OBS"]]
+
+## Setting Up Your Stream
+
+In your settings on Twitch, please enable VODs (video on demand) especially if you're working on OCF stuff.
+To do this, go to https://dashboard.twitch.tv/settings/channel and enable "Store past broadcasts".
+
+When streaming, keep the stream manager open (ideally on another screen) to see chat: https://dashboard.twitch.tv/stream-manager.
+
+## Setting up OBS to stream to Twitch
+
+OBS is pretty straightforward to configure. You can install it as you would any other software (arch pkg: obs-studio).
+
+Please record on OBS as well streaming! (Click both the "Start Streaming" and "Start Recording" buttons.)
+This will let us upload the streams to another source like YouTube. Twitch only saves VODs for a short period of time
+It's recommended to set up at least two scenes in OBS: one that shows your desktop and one that does not. This way, you can hide your screen if you need to work with passwords or otherwise sensitive stuff. Keep in mind that "it only showed for a split second" is not an excuse to not hide things. Twitch allows people to clip and rewatch anything frame-by-frame.
```

Comment:
good info but not sure the tone is what we should be going for ("is not an excuse")

Reply:
I don't want people to leak OCF secrets and be like "oh it's fine, I don't need to roll that password, it only showed for half a second!" I think that a lot of people have a mindset like that. That said, this can probably be reworded to be a little nicer and more clear.